connectors/http/src/main/java/quarks/connectors/http/runtime/HttpRequester.java

```diff
@@ -69,16 +84,35 @@ public R apply(T t) {
         String m = method.apply(t);
         String uri = url.apply(t);
         HttpUriRequest request;
+        
         switch (m) {
-        case HttpGet.METHOD_NAME:          
+        
+        case HttpGet.METHOD_NAME:
             request = new HttpGet(uri);
             break;
-        case HttpDelete.METHOD_NAME:          
+        case HttpDelete.METHOD_NAME:
             request = new HttpDelete(uri);
             break;
+        case HttpPost.METHOD_NAME:
+            request = new HttpPost(uri);
+            break;
+        case HttpPut.METHOD_NAME:
+            request = new HttpPut(uri);
```

Comment:
Adding Put seems like a good idea.  Of course that then begs the question: should there also be {put,delete}Json()? (and tests)  :-)